packages/ffe-accordion-react/src/AccordionItem.js

```diff
@@ -155,8 +141,7 @@ AccordionItem.propTypes = {
 AccordionItem.defaultProps = {
     hasNestedCollapse: false,
     ignoredNodeNames: [],
-    onClose: f => f,
-    onOpen: f => f,
+    onToggle: f => f,
```

Comment:
`Close` and `Open` give better hints about behavior than `Toggle` in my opinion. Can this function name mention what is being toggled?

Reply:
Maybe it's a given. Hmm. I'll let you decide :wink:

Reply:
You mean like `onToggleOpen`? Sure.